nemo/core/optim/optimizers.py

```diff
@@ -13,6 +13,7 @@
 # limitations under the License.
 
 import copy
+from apex.optimizers import FusedLAMB
```

Comment:
Add apex to our requirements

Reply:
Just to make sure: `apex` is baked into the base Docker container. Do you still want to specify it in `requirements.txt` ?

Reply:
Yes, not all users use docker

Reply:
Done